main.py

```diff
@@ -1,78 +1,31 @@
-from enum import Enum
-import argparse
-
+import typer
 import prediction_market_agent as pma
-from prediction_market_agent.tools.types import xdai_type
+from decimal import Decimal
+from prediction_market_agent.agents.all_agents import AgentType, get_agent
+from prediction_market_agent.tools.types import xDai, Mana
 from prediction_market_agent.tools.utils import should_not_happen, check_not_none
+from prediction_market_agent.markets.all_markets import MarketType, get_binary_markets
 
 
-class AgentType(Enum):
```

Comment:
I just moved these Enums to different files, so it's nicely importable in the agent entry point, too.